docs/Installation.md

```diff
@@ -26,10 +26,18 @@
   * [6.1 Smoke test](#61-smoke-test)
   * [6.2 What to do next?](#62-what-to-do-next)
 * [7. Upgrade Zonemaster database](#7-upgrade-zonemaster-database)
-* [Appendices](#appendices)

```

Comment:
I find it cleaner with the "Appendices".

Reply:
With the new solution there is one structure not two or three. There is one table of contents, not several.